uqcsbot/api.py

```diff
@@ -219,6 +222,16 @@ def _initialise(self):
             for page in self._bot.api.channels.list.paginate():
                 for chan in page['channels']:
                     self._add_channel(chan)
+
+            # The ims cover the direct messages between the bot and users
+            for page in self._bot.api.im.list.paginate():
+                for im in page['ims']:
+                    if im['is_user_deleted']:
```

Comment:
Is `is_user_deleted` guaranteed to be in the response?

Reply:
I believe so but I can replace it with a get if you'd like?

Reply:
Nah this is fine